src/locale/messages.en-US.xlf

```diff
@@ -486,14 +486,22 @@
         <source>keyboard-mapping-settings-page.delete-all-keyboard-mappings.success <x id="PH" equiv-text="actionTriggerIdsToBeDeleted.length"/></source>
         <target state="new">The <x id="PH" equiv-text="actionTriggerIdsToBeDeleted.length"/> selected keyboard mappings have been deleted.</target>
       </trans-unit>
-      <trans-unit id="6774239910313063167" datatype="html">
-        <source>keyboard-mapping.settings-page.invalid-field</source>
-        <target state="new">Invalid data, needs to be edited.</target>
-      </trans-unit>
       <trans-unit id="5805882241277564116" datatype="html">
         <source>keyboard-mapping-settings-page-import-keyboard-mappings.actions-missing</source>
         <target state="new">One or more keyboard mappings contained invalid actions and were not imported!</target>
       </trans-unit>
+      <trans-unit id="8483503986037571867" datatype="html">
+        <source>action-triggers.override-color.label</source>
+        <target state="new">Override default color</target>
```

Comment:
Since what _default color_ refers to is not specified anywhere, I suggest that we change it to `Override action color` or name it something like `Key color`.

Reply:
Update to override Key color

Reply:
Is it intentional to keep `default` in this translation?